.github/workflows/installer-script-test.yml

```diff
@@ -0,0 +1,71 @@
+name: installer-script-test
+
+on: 
+  push:
+    tags:
+      - '*'
+    paths:
```

Comment:
Let's also include this workflow file itself.

Reply:
Changes made.